src/App.css

```diff
@@ -27,14 +48,8 @@ li a {
   font-weight: 500;
 }
 
-.book {
-  display: flex;
-  flex-direction: row;
-}
-
-.book .action {
-  display: flex;
-  flex-direction: column;
-  gap: 2px;
-  width: 100%;
+.main {
```

Comment:
You're making great progress in your project, but it's crucial to ensure that the main class div adheres to the design specifications. Here are some specific updates to make:

- [ ] please update the background color of the main class div to match the design's color. Also please adjust the border radius of the div to match the design's specified value. Also please ensure that the box shadow of the div aligns with the design's shadow style. Also please update padding and margin to match the design.

By implementing these changes, you'll ensure that the main class div closely follows the design, resulting in a more visually consistent and appealing user interface. Keep up the excellent work! 🌟🎨